code/modules/mob/living/carbon/human/species/monkey.dm

```diff
@@ -38,6 +38,7 @@
 	total_health = 75
 	brute_mod = 1.5
 	burn_mod = 1.5
+	hunger_drain = HUNGER_FACTOR / 2 // twice has slow as normal
```

Comment:
```suggestion
	hunger_drain = HUNGER_FACTOR / 2 // twice as slow as normal
```

Reply:
i meant to put `as`, I swear
![minor-spelling-mistake](https://github.com/ParadiseSS13/Paradise/assets/91113370/ceb225c0-1fea-4128-9ac3-a3ef7e71b6d8)